raiden/tests/unit/test_pfs_integration.py

```diff
@@ -877,3 +878,48 @@ def test_post_pfs_feedback(query_paths_args):
         )
 
         assert not feedback.called
+
+
+def test_no_iou_when_pfs_price_0(query_paths_args):
+    """ Test that no IOU is sent when PFS is for free """
+    PFS_CONFIG_with_0_fee = PFSConfig(
```

Comment:
Can you remove the temporary variable here?

Reply:
yes, done